tests/kw_test.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-. ./tests/utils --source-only
+include "./tests/utils"
 unset -v KW_LIB_DIR  # to be able to test the developer mode
-. ./kw --source-only > /dev/null
+include "./kw" > /dev/null
```

Comment:
This looks like a corner case... at least document this behavior in the include function and the documentation.